src/SqlUtils.ts

```diff
@@ -80,8 +80,8 @@ export default class SqlUtils {
             };
         }
         catch (error) {
-            core.debug(`${error.message}`);
-            core.debug(`SqlCmd stderr: ${sqlCmdError}`);
+            console.log(`${error.message}`);
```

Comment:
Where does core.debug output to? Does changing this surface areas more easily?

Reply:
core.debug is only output if the user has selected the workflow to run with debug logging (https://docs.github.com/en/actions/monitoring-and-troubleshooting-workflows/enabling-debug-logging)

For most connectivity issue reports we're having to ask the users to go back and re-run the workflow with debug logging to get the basic connection failure information.  Changing this to console.log will output on any run that catches an error here.